README.rst

```diff
@@ -360,6 +360,10 @@ Choose from the list of available fixers:
                         Remove trailing commas in list
                         function calls.
 
+* **lower_case_cast** [symfony]
```

Comment:
hmm, you have renamed it to `lowercase_cast` but my proposal was `lowercase_casts`..